### DISCUSSION TITLE
Custom PDE

### DISCUSSION THREAD

**Discussion by GerasimovIV** [zwicker-group/py-pde#481] [Q&A]
Hello, thank you for your great work! I am trying to find the tool for solving pde system of type:

$$
\begin{cases}
\frac{\partial u}{\partial t} = f(u, v) \\
\frac{\partial v}{\partial y} = g(u, v) \\
\end{cases}
$$

Where u and v are functions: u=u(x, y, t) and v=v(x, y, t)
I have looked at all examples from your library and documentation for custom systems setup but I am still not sure is it possible to solve with py-pde my system or it is not. Can you comment? Or give some hints? (of course we suppose that all: boundary and initial condition for the existence and uniqueness of the solution are given)

**Comment by david-zwicker** [comment]
It might be difficult with this package, but this also depends on the boundary conditions and initial conditions. Could you please give more details?
